sensing/autoware_pointcloud_preprocessor/docs/distortion-corrector.md

```diff
@@ -48,3 +48,24 @@ ros2 launch autoware_pointcloud_preprocessor distortion_corrector.launch.xml
 
 - The node requires time synchronization between the topics from lidars, twist, and IMU.
 - If you want to use a 3D distortion corrector without IMU, please check that the linear and angular velocity fields of your twist message are not empty.
+- The node updates the per-point azimuth and distance values based on the undistorted XYZ coordinates when the input point cloud is in the sensor frame (not in the `base_link`) and the `update_azimuth_and_distance` parameter is set to `true`. The azimuth values are calculated using the `cv::fastAtan2` function.
+- Please note that updating the azimuth and distance fields increases the execution time by approximately 20%. Additionally, due to the `cv::fastAtan2` algorithm's accuracy of about 0.3 degrees, there is a **possibility of changing beam order for high azimuth resolution LiDAR**.
```

Comment:
Can you state clearly if 0.3 is the maximum error?
If possible, put the average and maximum error

Reply:
changing *the* beam order

otherwise it could mean of a "changing beam order", as in a property of the device

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/96800274df0de221da68fa94607a50af805bfdd6.

From the opencv website (https://docs.opencv.org/3.4/db/de0/group__core__utils.html#ga7b356498dd314380a0c386b059852270) , I couldn't know the average error.